lib/config.js

```diff
@@ -0,0 +1,13 @@
+import {config} from '@bedrock/web';
+
+config.credentialDisplay = {
+  registration: [/*
```

Comment:
Hmm, I haven't looked that closely at the config registration format, but if we've just got some array here we may (or may not) need to modify to a map structure to make deployment configuration changes easier. We can leave this for now, just noting that it may come up later and require a modeling change. This is something we can chat with @mattcollier about later if necessary.